openlibrary/templates/home/scroll-to-top.html

```diff
@@ -0,0 +1,44 @@
+$def with (test)
+<style>
+#myBtn {
+  display: none;
+  position: fixed;
+  bottom: 20px;
+  right: 30px;
+  z-index: 99;
+  font-size: 18px;
+  border: none;
+  outline: none;
+  background-color:#3b98fc;
+  color: white;
+  cursor: pointer;
+  padding: 15px;
+  border-radius: 4px;
+}
+
+#myBtn:hover {
+  background-color: #555;
+}
+</style>
+<button onclick="topFunction()"id="myBtn" title="Go to top">Top</button>
```

Comment:
This text needs to be internationalized.

Reply:
You mean to say, making it available in all other supported languages?

Reply:
How should I do it?

Reply:
You don't need to do any translation, just mark the text to be translated as described here: https://github.com/internetarchive/openlibrary/wiki/Frontend-Guide#internationalization-i18n---for-programmers

Reply:
Done :+1: